src/components/Album.css

```diff
@@ -0,0 +1,111 @@
+body{
+  background-color: black;
+  font-family: Helvetica, sans-serif;
+}
+/* ----------- Style for header ------------  */
+.header{
+  text-align: center;
+  color: #fff;
+}
+.header h1{
+  font-size: 20px;
+  margin-bottom: 35px;
+}
+/* ----------- Style for all albums ------------  */
+.allAlbums{
+  color: #fff;
+  display: grid;
+  grid-template-columns: 1fr ;
+  padding: 30px;
+}
+.eachAlbum{
+  padding: 25px;
+}
+
+/* ----------- Style for pictures and icons ---------- */
+.cover-container {
+position: relative;
+display: grid;
+place-items: center;
+}
+.cover-image{
+  height: auto;
+  max-width: 600px;
+  width: 100%;
+}
+.cover-container:hover .cover-image {
+  opacity: 0.5;
+  transition: all 0.35s ease-in-out; /* Makes the transition easier */
+
+}
+
+.icon{
+  width: 35px;
+}
+.icon-play{
+  width: 45px;
+}
+
+.icon:hover{
+  width: 40px;
+}
+.icon-play:hover{
```

Comment:
I personally think the effect would be even cooler if the difference between hover and normal was > 5px